src/styles/styles.js

```diff
@@ -2553,6 +2553,12 @@ const styles = (theme) => ({
         opacity: 0,
     },
 
+    invisiblePopover: {
+        position: 'absolute',
+        opacity: 0,
+        left: -9999,
```

Comment:
Hey ✋ 
Coming from https://github.com/Expensify/App/issues/56014, this has caused a layout bug, since invisiblePopover was still taking up some space 
This was resolved by adjusting it's vertical position using `top: -999` so it's outside the viewable area